api/test/h3-data/mocks/h3-data.mock.ts

```diff
@@ -39,6 +39,43 @@ export const h3DataMock = async (
   return h3data.save();
 };
 
+// Same as h3DataMock but without inserting 1000 value at table creation. Can probably replace h3DataMock
+export const h3AlternativeDataMock = async (
+  h3TableName: string,
+  h3ColumnName: string,
+  materialH3Data?: { [key: string]: number | null },
+  indicatorId?: string,
+  year: number = 2010,
+): Promise<H3Data> => {
+  const formattedTableName: string = snakeCase(h3TableName);
+  const formattedColumnName: string = camelCase(h3ColumnName);
+
+  await getManager().query(
+    `CREATE TABLE ${formattedTableName} (h3index h3index, "${formattedColumnName}" float4);`,
+  );
+
+  if (materialH3Data) {
+    let query = `INSERT INTO ${formattedTableName} (h3index, "${formattedColumnName}") VALUES `;
+    const queryArr = [];
+    for (const [key, value] of Object.entries(materialH3Data)) {
+      queryArr.push(`('${key}', ${value})`);
+    }
+    query = query.concat(queryArr.join());
+    await getManager().query(query);
+  }
+
+  const h3data = new H3Data();
+  h3data.h3tableName = formattedTableName;
+  h3data.h3columnName = formattedColumnName;
+  h3data.h3resolution = 6;
+  h3data.year = year;
+  if (indicatorId) {
+    h3data.indicatorId = indicatorId;
+  }
+
+  return h3data.save();
+};
+
```

Comment:
Since you added a more accurate fixture for a fake material h3 data, maybe we can just use that fixture on the main h3DataMock function (or the one you just created) instead of having both, which seems to do the same but with different fixtures?